src/lib/components/SequenceViewer.react.js

```diff
@@ -147,8 +148,18 @@ export default class SequenceViewer extends Component {
             legend: this.props.legend,
         };
 
+        const coverage = this.props.coverage;
+
+        for (let i = 0; i < coverage.length; i++) {
+            coverage[i].onclick = this.getOnClick(i);
+        }
```

Comment:
Is there a good reason for doing this directly on a prop vs. mapping onto new objects?

Reply:
Fixed in 79f3899!